src/scripts/main.js

```diff
@@ -22,7 +22,19 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const list = document.createElement('ul');
+
+  for (const text in data) {
+    const item = document.createElement('li');
+
+    item.textContent = text;
+
+    list.append(item);
+
+    createTree(item, data[text]);
```

Comment:
we can add `if` to avoid empty ul in DOM
<img width="323" alt="image" src="https://user-images.githubusercontent.com/70220667/226988203-1d85310b-e6f0-4e86-b1d0-c4cd2d42eb8c.png">